Updates tips of the round to exclude mentions of virologist, and include coroners and heretic paths instead. (#82923)
tgstation/tgstation: strings/tips.txt

As a Ghost, you can both start and join capture the flag games through the minigames menu, or by clicking on one of the team spawners, which can be found under the "Misc" section of the orbit menu.
As a Ghost, you can double click on just about anything to follow it. Or just warp around!
As a Ghost, you can see the inside of a container on the ground by clicking on it.

> As a Heretic, the Path of Ash focuses on stealth and disorientation, to get shed off them in aggressibely emblazing finale.

Reviewer: `to get shed off them in aggressibely emblazing finale`? Not sure what this is trying to say.

Author: that it stops beign stealthy and disorienting in favour of quite an agressive, overt, ring of fire.

> As a Heretic, the Path of Moon will drive people around you literally crazy - if you succeed, perhaps crazy enough to become your friends.

Reviewer: ```suggestion
As a Heretic, the Path of Moon will literally drive people around you crazy - perhaps crazy enough to become your allies should you succeed.
```

> As a Heretic, the Path of Lock is a tider's best friend, and can open many pathways. Including, eventually, ones beyond the veil...

Reviewer: ```suggestion
As a Heretic, the Path of Lock is an Assistant's best friend, and can open many pathways. Including ones beyond the veil...
```

> As a Heretic, the Path of Flesh allows you to make a proper army through ghouls and various summons. Through ascencion, you get to become a one-man-army yourself.

Reviewer: ```suggestion
As a Heretic, the Path of Flesh allows you to raise an army by summoning ghouls and monsters from beyond the veil. Through ascension, you become a one-man army yourself.
```

As a Heretic, the Path of Void makes people wish they could scream in the vast emptiness of space or have a chance at escaping from you. In the end, the storm takes all.

> As a Heretic, the Path of Blade rewards your actual abillity to not get killed in a fight, by making you better and better at it. Eventually, even, making others unable to cheat with stuns.

Reviewer: ```suggestion
As a Heretic, the Path of Blade rewards your ability to fight, by making you better and better at it. Though ascension, you can become an ultimate dueling juggernaut.
```

> As a Heretic, the Path of Rust is quite overt, but makes you able to shrug of a lot of damage as everything around you slowly decays into nothing but rot and rust.

Reviewer: ```suggestion
As a Heretic, the Path of Rust is quite overt, but allows you to shrug of a lot of damage as everything around you slowly decays into nothing but rot and rust.
```

> As a Heretic, the Path of Cosmos allows you to take rightfull ownership of the very space other, less worthy fleshlings thread on. And if they don't respect your status, calling in a friend from beyond should show them.

Reviewer: ```suggestion
As a Heretic, the Path of Cosmos allows you to take rightful ownership of the very space the crew treads on. And if they don't respect your status, calling in a friend from beyond should show them.
```

As a Medical Doctor, you can point your penlight at people to create a medical hologram. This lets them know that you're coming to treat them.
As a Medical Doctor, you can surgically implant or extract things from people's chests. This can range from putting in a bomb to pulling out an alien larva.
As a Medical Doctor, you must target the correct limb and not be in combat mode when trying to perform surgery on someone. Right clicking your patient will intentionally fail the surgery step.
As a Medical Doctor, when messing with viruses, remember that robotic organs can give immunity to disease effects and transmissibility. Make use of the inorganic biology symptom to bypass the protection.
As a Medical Doctor, while there's an pandemic, you only require small amounts of vaccine to heal a sick patient. Work with the Chemist to distribute your cures more efficiently.

> As a Medical Doctor, try messing with the virology lab sometime!  Viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!

Reviewer 2: ```suggestion
As a Medical Doctor, try messing with the virology lab sometime! Viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
```

As the AI, you can use CTRL + 1-9 to set a location hotkey for your camera, allowing you to save the location and jump to it at will. Tilde and zero will return you to the last spot you jumped from, and the numpad numbers act as aliases to the regular number keys.
As the Bartender, the drinks you start with only give you the basics. If you want more advanced mixtures, look into working with chemistry, hydroponics, or even mining for things to grind up and throw in!
As the Bartender, you can use a circular saw on your shotgun to make it easier to store.

> As the Bartender, remember to set up the bar sign by walking up to it and pressing on it!

Reviewer: ```suggestion
As the Bartender, remember to set up the bar sign by walking up to it and clicking it!
```

As the Clown, if you lose your banana peel, you can still slip people with your PDA! Honk!
As the Clown, if you're a Traitor and get an emag on sale (or convince another traitor), you can emag your Clown Car to unlock a variety of new functions, including the Siege Mode, which will allow you to launch your passengers, preferably directly into the Supermatter! Or into space.
As the Clown, spice your gimmicks up! Nobody likes a one-trick pony.
As the Clown, click with one long balloon in hand onto another to create a balloon animal! Each combination of colours has its own unique result.
As the Clown, you can use your stamp on a sheet of cardboard as the first step of making a honkbot. Fun for the whole crew!
As the Clown, your Grail is the mineral bananium, which can be given to the Roboticist to build you a fun and robust mech beloved by everyone.
As the Curator, be sure to keep the shelves stocked and the library clean for crew.

> As the Curator, you can be the menace of all felenids thanks to the laser pointer you start with. Unless, which is likely, you are a weeb, in which case; try a gentler approach.

Reviewer: ???? Just toss this tip out

As the Curator, you are not completely defenseless. Your whip easily disarms people, your laser pointer can blind humans and cyborgs, and you can hide items in wirecut books.

> As the Coroner, you are the medicaly certified edgelord of the station, which comes with several benefits, such as liking corpses, freely drinking medicaments meant for corpses such as formaldehyde and performing corpse-related surgeries faster.
> As the Coroner, your vendor contains appropriately edgy weaponry - you just need to hack in to get it, much like peforming an organ harvest.

Reviewer: I would toss both of these tips out and replace them with something that just covers their general "corpse surgery" niche
```suggestion
As the Coroner, you are more comfortable working on cadavers. You can perform autopsies or harvest organs from corpses a lot faster than your Medical Doctor counterparts. Work in tandem with them by helping prepare bodies for revival.
```

> As the Coroner, perform autopsies on corpses recovered from strange circumstances to discover just how they died! Teaming up with a detective, you can solve several cases together!

Reviewer: ```suggestion
As the Coroner, you can perform autopsies on corpses recovered from strange circumstances with your handheld autopsy scanner to discover how they died. By teaming up with a Detective, you can solve several cases together!
```
